src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -334,6 +335,12 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Throttling.TimeIntervalInMs", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public TimeSetting? ThrottlingTimeInterval { get; protected set; }
 
+        [Description("Index engine for AutoIndexes")]
+        [DefaultValue(Documents.Indexes.IndexingEngine.Corax)]
+        [IndexUpdateType(IndexUpdateType.Reset)]
+        [ConfigurationEntry("Indexing.Auto.Engine", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
+        public IndexingEngine? AutoIndexingEngine { get; protected set; }
```

Comment:
I think `SearchEngine` is so commonly used and recognized that it would suit better than `IndexingEngine`. Then also the configuration name would changed to `Indexing.Auto.SearchEngine`